### DISCUSSION TITLE
Download files mp3

### DISCUSSION THREAD

**Discussion by axaaxa6520** [rejetto/hfs#429] [General]
Congratulations on the project. I'm trying this file server to manage my mp3 files.
I wanted to know if it is possible to use one of the many download programs (e.g. idm, fdm, jdownloader, etc.) to automatically download all the contents of the folder at the same time, and possibly use wildcards because the folder title is part of the folder of the file is a date that changes every day.
I tried to do some tests but the result was to have a file called index.html

**Comment by rejetto** [comment]
hey, I don't know what are the capabilities of those programs, I know about HFS.
One possibility to download a whole folder is to get it zipped.
You have the zip button on the top-right corner, but if you need an URL then it's just the folder with "?get=zip" appended.
I didn't get the part where you talk about the wildcards. It sounds like you are trying to automate something, but that heavilty depends on your needs and your skills. Surely there can be multiple solutions, but you can probably arrange things in a way that makes the job easier, but to do that one must know exactly what you need.
For example, you say there's a changing date every day, but if the date you are talking about is today's date then you may even not need wildcards, as the command to download could form the address using the correct date.
If I had to automate something I would probably consider command line tools like wget.
Also, I would consider describing the problem to GPT or Bing, you may get good hints. This is just an example:
<img width="721" alt="image" src="https://github.com/rejetto/hfs/assets/1367199/307c0a3f-1e6f-4c24-992c-fe2f014ac74e">
